DFC.ServiceTaxonomy.Taxonomies/Helper/TaxonomyHelpers.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using DFC.ServiceTaxonomy.Taxonomies.Models;
+using OrchardCore.ContentManagement;
+
+namespace DFC.ServiceTaxonomy.Taxonomies.Helper
+{
+    public static class TaxonomyHelpers
```

Comment:
It'd be good if we can avoid static helper classes, as they make unit testing harder.